definitions/npm/lodash_v4.x.x/flow_v0.47.x-/test_lodash-v4.x.x.js

```diff
@@ -0,0 +1,348 @@
+  stringsGroupedByLength[3][0].toLowerCase();
+}
+var numbersObj: {[key: string]: number} = {a: 6.1, b: 4.2, c: 6.3};
+var numbersGroupedByMathFloor2 = _.groupBy(numbersObj, Math.floor);
+if (numbersGroupedByMathFloor2[6]) {
+  numbersGroupedByMathFloor2[6][0] / numbersGroupedByMathFloor2[6][1]
+}
+var stringObj: {[key: string]: string} = {a: 'one', b: 'two', c: 'three'};
+var stringsGroupedByLength2 = _.groupBy(stringObj, 'length');
+if (stringsGroupedByLength2[3]) {
+  stringsGroupedByLength2[3][0].toLowerCase();
+}
+
+
+/**
+ * _.intersectionBy
+ */
+_.intersectionBy([2.1, 1.2], [2.3, 3.4], Math.floor);
+_.intersectionBy([{ 'x': 1 }], [{ 'x': 2 }, { 'x': 1 }], 'x');
+
+
+/**
+ * _.get
+ */
+
+// Object — examples from lodash docs
+var exampleObjectForGetTest = { 'a': [{ 'b': { 'c': 3 } }] };
+_.get(exampleObjectForGetTest, 'a[0].b.c');
+_.get(exampleObjectForGetTest, ['a', '0', 'b', 'c']);
+_.get(exampleObjectForGetTest, 'a.b.c', 'default');
+
+// Array — not documented, but _.get does support arrays
+_.get([1, 2, 3], '0');
+_.get(['foo', 'bar', 'baz'], '[1]');
+_.get([{ a: 'foo' }, { b: 'bar' }, { c: 'baz' }], '2');
+_.get([[1, 2], [3, 4], [5, 6], [7, 8]], '3');
+
+// Second argument must be string when looking for array items by index
+// $ExpectError number This type is incompatible with union: ?array type | string
+_.get([1, 2, 3], 0);
+
+
+/**
+ * _.keyBy
+ */
+_.keyBy([
+  { 'dir': 'left', 'code': 97 },
+  { 'dir': 'right', 'code': 100 }
+], function(o) {
+  return String.fromCharCode(o.code);
+});
+_.keyBy([
+  { 'dir': 'left', 'code': 97 },
+  { 'dir': 'right', 'code': 100 }
+], 'dir');
+
+
+/**
+ * _.map examples from the official doc
+ */
+function square(n) {
+  return n * n;
+}
+
+_.map([4, 8], square);
+_.map({ 'a': 4, 'b': 8 }, square);
+
+var users = [
+  { 'user': 'barney' },
+  { 'user': 'fred' }
+];
+
+// The `_.property` iteratee shorthand.
+_.map(users, 'user');
+
+
+/**
+ * _.pullAllBy
+ */
+_.pullAllBy([{ 'x': 1 }, { 'x': 2 }, { 'x': 3 }, { 'x': 1 }], [{ 'x': 1 }, { 'x': 3 }], 'x');
+
+
+/**
+ * _.unionBy
+ */
+_.unionBy([2.1], [1.2, 2.3], Math.floor);
+_.unionBy([{ 'x': 1 }], [{ 'x': 2 }, { 'x': 1 }], 'x');
+
+
+/**
+ * _.uniqBy
+ */
+_.uniqBy([2.1, 1.2, 2.3], Math.floor);
+_.uniqBy([{ 'x': 1 }, { 'x': 2 }, { 'x': 1 }], 'x');
+
+
+/**
+ * _.clone
+ */
+_.clone({a: 1}).a == 1;
+// $ExpectError property `b`. Property not found in object literal
+_.clone({a: 1}).b == 1
+// $ExpectError number. This type is incompatible with function type.
+_.clone({a: 1}).a == 'c';
+
+/**
+ * _.isEqual
+ */
+_.isEqual('a', 'b');
+_.isEqual({x: 1}, {y: 2});
+
+// Flow considers this compatible with isEqual(a: any, b: any).
+// Reasonable people disagree about whether this should be considered a legal call.
+// See https://github.com/splodingsocks/FlowTyped/pull/1#issuecomment-149345275
+// and https://github.com/facebook/flow/issues/956
+_.isEqual(1);
+
+// $ExpectError function type expects no more than 2 arguments
```

Comment:
This is only addition to this test file, otherwise is identical to the Lodash test file for `flow_v0.38.x-v0.46.x`